src/models/generated/DemandsideTechnology.jl

```diff
@@ -6,98 +6,124 @@ This file is auto-generated. Do not edit.
 
 """
     mutable struct DemandSideTechnology{T <: PSY.StaticInjection} <: Technology
-        min_power::Float64
+        price_per_unit::Float64
```

Comment:
what are these ramp percentages? These are operational parameters that don't exist in PowerSystems.jl